examples/ota-provider-app/linux/main.cpp

```diff
@@ -50,31 +50,117 @@ using chip::Messaging::ExchangeManager;
 constexpr chip::EndpointId kOtaProviderEndpoint = 0;
 
 constexpr uint16_t kOptionFilepath             = 'f';
+constexpr uint16_t kOptionOtaImageList         = 'o';
 constexpr uint16_t kOptionQueryImageBehavior   = 'q';
 constexpr uint16_t kOptionDelayedActionTimeSec = 'd';
 
 // Global variables used for passing the CLI arguments to the OTAProviderExample object
-OTAProviderExample::QueryImageBehaviorType gQueryImageBehavior = OTAProviderExample::kRespondWithUpdateAvailable;
-uint32_t gDelayedActionTimeSec                                 = 0;
-const char * gOtaFilepath                                      = nullptr;
+static OTAProviderExample::QueryImageBehaviorType gQueryImageBehavior = OTAProviderExample::kRespondWithUpdateAvailable;
+static uint32_t gDelayedActionTimeSec                                 = 0;
+static const char * gOtaFilepath                                      = nullptr;
+static const char * gOtaImageListFilepath                             = nullptr;
+
+// Basic CSV parser that expects a precise format - Device Software Version Model Schema.
```

Comment:
I wonder if it makes sense to add this to the README instead of here.

Reply:
Good point. I will update the README with the required information.